internal/db/schema/migrations/postgres/2/01_job.up.sql

```diff
@@ -0,0 +1,87 @@
+begin;
+
+create table job (
+     private_id wt_private_id primary key,
+     name wt_name not null,
+     description wt_description not null,
+     code text not null
+         constraint code_too_short
+             check (length(trim(value)) > 0)
+         constraint code_too_long
+             check (length(trim(value)) < 128),
```

Comment:
I'm getting the following error when I run the migration:

```shell
psql:internal/db/schema/migrations/postgres/2/01_job.up.sql:16: ERROR:  column "value" does not exist
LINE 7:              check (length(trim(value)) > 0)
```

Reply:
grrr I fixed this locally but clearly never pushed, ty